docs/content/Modpacks/Recipes/Custom-Recipe-Modifiers.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Custom Recipe Modifiers
+---
+
+# Custom Recipe Modifiers / Data Logic
+
+## Adding Modifier
+Custom recipe modifiers in KubeJS are done through a function. For an example, we will make multiblock that requires temperature for recipes. (Like EBF does)
+```js title="temperature_recipe_modifier.js"
+const $GTRecipe = Java.loadClass("com.gregtechceu.gtceu.api.recipe.GTRecipe");
+const $MetaMachine = Java.loadClass("com.gregtechceu.gtceu.api.machine.MetaMachine");
+
+function TemperatureModifier(machine, recipe) {
+    if (!(machine instanceof $MetaMachine)) return ModifierFunction.NULL
+    if (!(recipe instanceof $GTRecipe)) return ModifierFunction.NULL
+    
+    if (!machine instanceof $CoilWorkableElectricMultiblockMachine) {
+        return $RecipeModifier.nullWrongType($CoilWorkableElectricMultiblockMachine, machine);
+    } else {
+
+        let Temp = machine.getCoilType().getCoilTemperature() // (3)
+
+        let recipeTemp = recipe.data.getInt("RequiredTemp") // (4)
+        if (!(Temp > recipeTemp)) {
```

Comment:
```suggestion
        if (recipeTemp > temp) {
```

Reply:
Also this would have broken if temp == recipeTemp which isn't inteded behavior I assume